client/ayon_deadline/repository/custom/plugins/GlobalJobPreLoad.py

```diff
@@ -496,6 +496,12 @@ def inject_ayon_environment(deadlinePlugin):
             "extractenvironments",
             export_url
         ]
+
+        # staging requires passing argument
+        settings_variant = job.GetJobEnvironmentKeyValue("AYON_DEFAULT_SETTINGS_VARIANT")  # noqa
```

Comment:
This should not be needed. It is in environment variables...

Reply:
Maybe we should just add it to `environment`, instead of modifying arguments.

Reply:
It was not working if I set it via process `AYON_DEFAULT_SETTINGS_VARIANT`. I was looking into launcher and it seemed it was looking only for argument and not env.

Dev and production is working correctly.

Reply:
Marked with TODO